releases.json

```
@@ -2343,6 +2343,14 @@
       "1.24-1"
     ]
   },
+  "websocketpp": {
+    "dependency_names": [
+      "websocketpp"
+    ],
+    "versions": [
+      "0.8.2-0"
```

```suggestion
      "0.8.2-1"
```

It's the first published wrap for this websocketpp release.

---

Ah, fdk-aac and a couple of other wraps use `-0`. Fixed.

---

It's a very wishy-washy thing to begin with. I just tend to view "0" as meaning "locally testing this, will bump for publication", so it looks off. :D